src/test/ui/moves/move-out-of-slice-2.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(unsized_locals)]
+//~^ WARN the feature `unsized_locals` is incomplete and may cause the compiler to crash
```

Comment:
That message doesn't really capture "or produce wrong code".

I am changing the message to something more appropriate for cases like this in https://github.com/rust-lang/rust/pull/71420, but that PR is seemingly stuck unfortunately.

Reply:
I'll make a PR with just the message change.